borg/archive.py

```diff
@@ -298,35 +298,16 @@ def add_file_chunks(chunks):
         cache.rollback()
```

Comment:
"This over avoids storing all files in a mapping and allows proper preloading for new archives."

Oops. Guess I over-pasted part of the commit message :D